server/config/toml.go

```diff
@@ -237,9 +237,31 @@ snapshot-interval = {{ .StateSync.SnapshotInterval }}
 snapshot-keep-recent = {{ .StateSync.SnapshotKeepRecent }}
 
 ###############################################################################
-###                         Store / State Streaming                         ###
+###                              State Streaming                            ###
 ###############################################################################
 
+# Streaming allows nodes to stream state to external systems
+[streaming]
+
+# streaming.abci specifies the configuration for the ABCI Listener streaming service
+[streaming.abci]
+
+# List of kv store keys to stream out via gRPC.
+# The store key names MUST match the module's StoreKey name.
+#
+# Example:
+# ["acc", "bank", "gov", "staking", "mint"[,...]]
+# ["*"] to expose all keys.
+keys = [{{ range .Streaming.ABCI.Keys }}{{ printf "%q, " . }}{{end}}]
+
+# The plugin name used for streaming via gRPC.
+# Supported plugins: abci_v1
+plugin = "{{ .Streaming.ABCI.Plugin }}"
+
+# stop-node-on-err specifies whether to stop the node on message delivery error
+stop-node-on-err = {{ .Streaming.ABCI.StopNodeOnErr }}
+
+# Legacy streaming support
 [store]
 streamers = [{{ range .Store.Streamers }}{{ printf "%q, " . }}{{end}}]
```

Comment:
This needs to be removed along with the config objects related to it. None of this stuff is used or functional anymore.

Reply:
removed.